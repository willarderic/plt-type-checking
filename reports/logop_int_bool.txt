TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: OR  LEXEME: or						TOKEN: OR        LEXEME: or
LOG_OP -> or							LOG_OP -> or
BIN_OP -> LOG_OP						BIN_OP -> LOG_OP
TOKEN: INTCONST  LEXEME: 1					TOKEN: INTCONST  LEXEME: 1
CONST -> INTCONST						CONST -> INTCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: NIL  LEXEME: nil						TOKEN: NIL       LEXEME: nil
CONST -> nil							CONST -> nil
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR							START -> EXPR

---- Completed parsing ----					---- Completed parsing ----

