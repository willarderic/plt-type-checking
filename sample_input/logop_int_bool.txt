(or 1 nil)

