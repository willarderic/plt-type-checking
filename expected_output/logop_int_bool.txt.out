TOKEN: LPAREN    LEXEME: (
TOKEN: OR        LEXEME: or
LOG_OP -> or
BIN_OP -> LOG_OP
TOKEN: INTCONST  LEXEME: 1
CONST -> INTCONST
EXPR -> CONST
TOKEN: NIL       LEXEME: nil
CONST -> nil
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR

---- Completed parsing ----

